src/ApiCompat/Microsoft.DotNet.ApiCompat.Task/SuppressibleMSBuildLog.cs

```diff
@@ -7,16 +7,16 @@
 namespace Microsoft.DotNet.ApiCompat.Task
 {
     /// <summary>
-    /// Class that can log Suppressions in an MSBuild task, by implementing MSBuildLog and ISuppressableLog.
+    /// Class that can log Suppressions in an MSBuild task, by implementing MSBuildLog and ISuppressibleLog.
     /// </summary>
-    internal sealed class SuppressableMSBuildLog : MSBuildLog, ISuppressableLog
+    internal sealed class SuppressibleMSBuildLog : MSBuildLog, ISuppressableLog
```

Comment:
```suggestion
    internal sealed class SuppressibleMSBuildLog : MSBuildLog, ISuppressibleLog
```
?

Reply:
Oh golly. Okay. Let me get all those too.

Reply:
Fixed all the uses of 'suppressable' in the repo.